.github/workflows/update-gradle-wrapper.yml

```diff
@@ -0,0 +1,18 @@
+name: Update Gradle Wrapper
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Can you please align this with the dependabot PRs? I think it was weekly, Monday night.

Reply:
This wasn't that easy, because it's not GitHub actiosn anymore.

The documentation https://dependabot.com/docs/config-file/ is not that useful. One has to go to https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates#scheduletime to find out that it happens on `05:00 UTC`.